CHANGELOG.md

```diff
@@ -17,6 +17,9 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+### BREAKING CHANGES
+- added `box-sizing: border-box` to all elements, as well as before and after pseudo elements in Teams theme @mnajdova ([#1057](https://github.com/stardust-ui/react/pull/1057))
```

Comment:
nit-nit: lets capitalize this entry, as well as provide it in the Present tense: `Add box-sizing: border-box to ..` - just to be consistent with CHANGELOG contents